spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/repository/query/SpannerStatementQueryExecutor.java

```diff
@@ -162,6 +164,9 @@ private SpannerStatementQueryExecutor() {
 							.getPersistentProperty(o.getProperty());
 					return (property != null) ? property.getColumnName() : o.getProperty();
 				});
+		if (!persistentEntity.getWhere().isEmpty()) {
```

Comment:
This construction `persistentEntity.getWhere().isEmpty()` is used in multiple places.  Please create `hasWhere()` method to improve readability.